empyrical/tests/test_stats.py

```diff
@@ -71,24 +71,24 @@ class TestStats(TestCase):
     # Flat line
     flat_line_0 = pd.Series(
         np.linspace(0, 0, num=1000),
-        index=pd.date_range('2000-1-30', periods=1000, freq='D')
+        index=pd.date_range('2000-1-30', periods=1000, freq='D', tz='UTC')
     )
     # Flat line
     flat_line_1 = pd.Series(
         np.linspace(0.01, 0.01, num=1000),
-        index=pd.date_range('2000-1-30', periods=1000, freq='D')
+        index=pd.date_range('2000-1-30', periods=1000, freq='D', tz='UTC')
```

Comment:
Sorry, I meant in addition to non-UTC. Just a single one with UTC and arbitrary data.